src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -14,10 +14,9 @@
        href="{{ docsUrl }}"
        target="_blank"
        i18n>documentation</a>
-    <a ngbDropdownItem
-       href="/docs"
-       target="_blank"
```

Comment:
The previous behavior was to open the API docs page in a new tab, now is loading in the same tab so you are out of the Dashboard.

Reply:
Updated